Jenkinsfile

```diff
@@ -224,7 +225,8 @@ node ('master') {
         echo "All Default"
         break;
      case 'Before merge to trunk':
-        gitNotify ("Jenkins: Merge to trunk", "Started...", 'PENDING')
```

Comment:
The wrong line, you need to disable in  `'Commit in Open PR':` section  (4 line up)

Reply:
done